### PR TITLE
added a new endpoint to userstore, get user by uid

### PR DESCRIPTION
**Pull request by iljaSL**
Added a new endpoint to userstore, get user by UID.

### CODE REVIEW

File: api/userstore/client.go [hunk 1]
```diff
@@ -26,6 +28,17 @@ func New(api restapi.Connector) *UserStore {
 	return &UserStore{api: api}
 }
 
+// GetUser get user details by user ID
+func (store *UserStore) GetUser(id string) (user *LocalUser, err error) {
```
Comment (fogfish) [inline api/userstore/client.go:32]: Please consider a best practice of Golang naming https://golang.org/doc/effective_go#Getters

Role Store API already have methods to get Users. It gonna be confusing to have both.

**Comment by iljaSL** [comment]
I also changed the function name from a previous merged branch, from `GetLocalUsers` to `LocalUsers`.
I hope it's okay to do the change in the current branch.

File: api/userstore/client.go [hunk 2]
```diff
@@ -26,6 +28,17 @@ func New(api restapi.Connector) *UserStore {
 	return &UserStore{api: api}
 }
 
+// LocalUser returns details about the local user
+func (store *UserStore) LocalUser(id string) (user *LocalUser, err error) {
```
Comment (kimmokssh) [inline api/userstore/client.go:32]: Would it make sense to keep the Get, Create, Update and Delete for the function names? Then it is clear what the function does.

Comment (fogfish) [reply api/userstore/client.go:32]: Usage of Get is against best Golang practice: https://golang.org/doc/effective_go#Getters but all in all this your call. May be this is a time to make decision and alight it everywhere.

**Review by fogfish — APPROVED** [review]

**Pull request merged**